### DISCUSSION TITLE
Eclipse predictions

### DISCUSSION THREAD

**Discussion by margre8** [Stellarium/stellarium#217] [Q&A] [answered]
Greetings,
can I in Stellarium predict an eclipse of Sun and Moon?

Thank you

**Comment by gzotti** [comment]
Yes.

**Comment by margre8** [comment]
Thank you,
and how can I do it? (because I did not find it this option).

**Comment by gzotti** [comment]
- Set date of eclipse.
- center on eclipsed body.

**Comment by margre8** [comment]
OK.
And is this option in astronomical calculations window too?
Thanks for answer.

**Comment by gzotti** [comment, marked as answer]
Seems I misunderstood the question. There is no "Find next eclipse"-style prediction, but you can predict its approximate appearance when you input the time.

**Comment by margre8** [comment]
Thank you.
I already understand.
